sos/plugins/ovn_central.py

```diff
@@ -66,10 +99,20 @@ def setup(self):
 
         schema_dir = '/usr/share/openvswitch'
 
-        self.add_database_output(os.path.join(schema_dir, 'ovn-nb.ovsschema'),
-                                 cmds, 'ovn-nbctl')
-        self.add_database_output(os.path.join(schema_dir, 'ovn-sb.ovsschema'),
-                                 cmds, 'ovn-sbctl', ['Logical_Flow'])
+        nb_tables = self.get_tables_from_schema(os.path.join(
+            schema_dir, 'ovn-nb.ovsschema'))
+        sb_tables = self.get_tables_from_schema(os.path.join(
+            schema_dir, 'ovn-sb.ovsschema'), ['Logical_Flow'])
+
+        self.add_database_output(nb_tables, cmds, 'ovn-nbctl')
+        self.add_database_output(sb_tables, cmds, 'ovn-sbctl')
```

Comment:
Will be "cmds" variable really updated in the `cmds.append('%s list %s' % (ovn_cmd, table))` call inside `add_database_output`? As:

    >>> def plus(a,b,c):
    ...   c = a+b
    ... 
    >>> s1=3
    >>> s2=5
    >>> s=6
    >>> plus(s1,s2,s)
    >>> s
    6

(this could be over-worked by using "self.cmds" in `setup` and `add_database_output` and not passing "cmds" as its argument)

Reply:
As it is a list, we can be sure that the cmds variable will be updated. (My patch is not changing the behavior though, it was like that before). For example:

def _bb(param):
    param.append(3)

aa = [1, 2]
_bb(aa)
print(aa)

$ python3.7 test.py
[1, 2, 3]

$ python2.7 test.py
[1, 2, 3]

Reply:
Ah right, forgot it is a list..